VSRAD.Package/DebugVisualizer/VisualizerTable.cs

```diff
@@ -248,7 +250,6 @@ private void SetupColumns()
                 HeaderText = "Name",
                 ReadOnly = false,
                 Frozen = true,
-                AutoSizeMode = DataGridViewAutoSizeColumnMode.AllCells,
```

Comment:
This disables the old behavior of the column automatically resizing to fit the data, even if `Options.VisualizerAppearance.ScaleNameColumn` is set to false. Is it intentional?

While the manual mode is nice to have, personally I find it less convenient compared to what we have now. Perhaps when `ScaleNameColumn` is disabled, we should keep the old behavior _and_ also autoresize the Name column when it's being edited, like this:

![autoresize](https://user-images.githubusercontent.com/32609527/181758625-ef6aa6ce-320b-42e6-af50-33818e6f1174.gif)

Because if I recall correctly not having this behavior was one of the reasons why we wanted a manual scaling mode in the first place.

Reply:
Thank you! Done in 70b0b27